src/main/resources/schema.sql

```diff
@@ -1,13 +1,43 @@
-DROP TABLE IF EXISTS hmhb_user;
+DROP TABLE IF EXISTS hmhb_user; /* get rid of this in the next PR */
 
-CREATE TABLE hmhb_user (
-    id SERIAL PRIMARY KEY,
-    email TEXT,
-    first_name TEXT,
-    last_name TEXT,
-    birth_date DATE
+DROP TABLE IF EXISTS requested_program;
+DROP TABLE IF EXISTS published_program;
+
+CREATE TABLE published_program (
+    id BIGSERIAL PRIMARY KEY,
+    organization_name TEXT NOT NULL,
```

Comment:
I've made a guess at what is nullable and what needs to be validated.  We can add to the validation (and change existing) after the scaffolding gets in and we've had more conversations with the client.